build.gradle

```diff
@@ -34,6 +34,18 @@ configurations.all {
     transitive = true
 }
 
+task checkAll(type: GradleBuild) {
+    tasks = ['pmd', 'checkstyle', 'lintDebug', 'spotbugsDebug', 'testDebugUnitTest', 'connectedDebugAndroidTest']
```

Comment:
you could use here just `checkCode` and `checkTest`

Reply:
You mean that we don't need `checkAll`, just typing `./gradew checkCode checkTest` ?

Reply:
I men you can use tasks = ['checkCode', 'checkTest']
instead of tasks = ['pmd', 'checkstyle', 'lintDebug', 'spotbugsDebug', 'testDebugUnitTest', 'connectedDebugAndroidTest']

Reply:
Good idea.